### PR TITLE
fix: race condition loading GWT JS [BACKLOG-40418]

### PR DESCRIPTION
**Pull request by afrjorge**
@pentaho/millenniumfalcon please review

### CODE REVIEW

File: user-console/src/main/java/org/pentaho/mantle/client/MantleUtils.java [hunk 1]
```diff
@@ -94,6 +94,10 @@ public boolean checkSelectedPerspective() {
   public static native String getSchedulerPluginContextURL()/*-{
     if (typeof $wnd.pho.getSchedulerPluginContextURL !== "undefined" ) {
       return $wnd.pho.getSchedulerPluginContextURL();
+    } else {
+      // fallback to the default location when the getter is unavailable
+      // e.g. race condition loading GWT JavaScript code, PUC JS executes before the Scheduler Plugin JS is loaded
+      return $wnd.location.protocol + "//" + $wnd.location.host + $wnd.CONTEXT_PATH + "plugin/scheduler-plugin/";
```
Comment (davidmsantos90) [inline user-console/src/main/java/org/pentaho/mantle/client/MantleUtils.java:100]: `FULL_QUALIFIED_URL` should also be available to use here. please use it instead, as it is equivalent to `$wnd.location.protocol + "//" + $wnd.location.host + $wnd.CONTEXT_PATH`

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 36s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5617/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 20s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5617/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by davidmsantos90 — APPROVED** [review]

**Review by mbrasil — APPROVED** [review]

**Pull request merged**